### PR TITLE
update doc cv32 with new debug spec for breakpoint

### PR DESCRIPTION
**Pull request by davideschiavone**

### CODE REVIEW

File: cores/cv32e40p/user_manual/source/control_status_registers.rst [hunk 1]
```diff
@@ -451,3 +466,141 @@ Reset Value: 0x0000_0000
...
+
+Accessible in Debug Mode or M-Mode when trigger support is enabled (using the DbgTriggerEn parameter).
+Since native triggers are not supported, writes to this register from M-Mode will be ignored.
+
+CV32E40P only implements one type of trigger, Match Control. Most fields of this register will read as a fixed value to reflect the single mode that is supported, in particular, instruction address match as described in the Debug Specification 0.13.2 section 5.2.2 & 5.2.9.
+
+
++-------+------+------------------------------------------------------------------+
+| Bit#  | R/W  | Description                                                      |
++-------+------+------------------------------------------------------------------+
```
Comment (MikeOpenHWGroup) [inline cores/cv32e40p/user_manual/source/control_status_registers.rst:498]: In RST tables, the line under the header is indidcated using "=".   So line 498 is should be:
`+=======+=======+===============================================+`
The "+" characters needs to line up with the "|" characters on the preceding line.

File: cores/cv32e40p/user_manual/source/control_status_registers.rst [hunk 2]
```diff
@@ -451,3 +466,141 @@ Reset Value: 0x0000_0000
...
+
+Reset Value: 0x0000_0000
+
+Accessible in Debug Mode or M-Mode when trigger support is enabled (using the DbgTriggerEn parameter). Since native triggers are not supported, writes to this register from M-Mode will be ignored.
+
+This register stores the instruction address to match against for a breakpoint trigger.
+
++-------+------+------------------------------------------------------------------+
+| Bit#  | R/W  | Description                                                      |
++-------+------+------------------------------------------------------------------+
```
Comment (MikeOpenHWGroup) [inline cores/cv32e40p/user_manual/source/control_status_registers.rst:548]: Same comment as above.

File: cores/cv32e40p/user_manual/source/control_status_registers.rst [hunk 3]
```diff
@@ -266,21 +290,15 @@ CSR Address: 0x342
...
 
-+-----------+----+----+----+---+
-| 31 : 4    | 3  | 2  | 1  | 0 |
-+===========+====+====+====+===+
-| Interrupt | Exception Code   |
-+-----------+------------------+
++-------------+-----------+----------------------------------------------------------------------------------+
+|   Bit #     |   R/W     |   Description                                                                    |
++=============+===========+==================================================================================+
++-------------+-----------+----------------------------------------------------------------------------------+
```
Comment (MikeOpenHWGroup) [inline cores/cv32e40p/user_manual/source/control_status_registers.rst:296]: This line is redundant.

File: cores/cv32e40p/user_manual/source/control_status_registers.rst [hunk 4]
```diff
@@ -451,3 +466,141 @@ Reset Value: 0x0000_0000
...
+
+Reset Value: 0x0000_0000
+
+Accessible in Debug Mode or M-Mode when trigger support is enabled (using the DbgTriggerEn parameter).
+
+CV32E40P does not support the features requiring this register. Writes are ignored and reads will always return zero.
+
++-------+------+------------------------------------------------------------------+
+| Bit#  | R/W  | Description                                                      |
++-------+------+------------------------------------------------------------------+
```
Comment (MikeOpenHWGroup) [inline cores/cv32e40p/user_manual/source/control_status_registers.rst:567]: Same comment: need to use "=" on line 567.

File: cores/cv32e40p/user_manual/source/control_status_registers.rst [hunk 5]
```diff
@@ -451,3 +466,141 @@ Reset Value: 0x0000_0000
...
```
Comment (MikeOpenHWGroup) [inline cores/cv32e40p/user_manual/source/control_status_registers.rst:586]: Same comment.

File: cores/cv32e40p/user_manual/source/control_status_registers.rst [hunk 6]
```diff
@@ -451,3 +466,141 @@ Reset Value: 0x0000_0000
...
```
Comment (MikeOpenHWGroup) [inline cores/cv32e40p/user_manual/source/control_status_registers.rst:604]: Same comment.

**Comment by davideschiavone** [comment]
Thanks @MikeOpenHWGroup !

I update my branch with another commit https://github.com/openhwgroup/core-v-docs/pull/30/commits/22d7b5307ebc8a9ea6264064aff1b706456eb6af

**Pull request merged**